lua/starfall/libs_cl/render.lua

```diff
@@ -527,6 +528,25 @@ function render_library.clearStencil()
 	render.ClearStencil()
 end
 
+--- Sets up the ambient lighting for any upcoming render operation. Ambient lighting can be seen as a cube enclosing the object to be drawn, each of its faces representing a directional light source that shines towards the object.
+-- @param number lightDirection The light source to edit.
```

Comment:
Can you mention the 'BOX' enum?

Reply:
There are more method docs that need this change, not a huge problem though.